### PR TITLE
Add virtual memory extension schema and example.

### PR DESCRIPTION
**Pull request by changab**

### CODE REVIEW

**Comment by cento** [comment]
looks fine to me

**Comment by daniellustig** [comment]
LGTM, thanks!

File: schema/virtual-memory-extension.asn [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Virtual-Memory-Extension
+DEFINITIONS
+   AUTOMATIC TAGS ::=
+BEGIN
+   EXPORTS VirtMem;
+
+   VirtMem ::= SEQUENCE {
+      -- 1 bit to indicate Svnapot standard extension is supported on Sv39, Sv48
+      -- and Sv57 page table entry.
+      svnapot BOOLEAN,
```
Comment (timsifive) [inline schema/virtual-memory-extension.asn]:
```suggestion
      svnapot BOOLEAN DEFAULT FALSE,
```

This way people can omit this bit in the description if it's not implemented.

The same for the two BOOLEANs below.

File: schema/virtual-memory-extension.asn [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+      -- 1 bit to indicate Svpbmt standard extension is supported on Sv39, Sv48
+      -- and Sv57 page table entry.
+      svpbmt  BOOLEAN,
+
+      -- 1 bit to indicate Svinval Standard Extension is supported.
+      svinval BOOLEAN
+   }
```
Comment (timsifive) [inline schema/virtual-memory-extension.asn:19]: We should end this SEQUENCE with `, ...` unless we're absolutely sure there will never be any more information from the virtual memory group.

**Review by timsifive — CHANGES_REQUESTED** [review]

File: schema/virtual-memory-extension.asn (hunk 1 above)
Comment (changab) [reply schema/virtual-memory-extension.asn]: good point. done in the second commit

File: schema/virtual-memory-extension.asn (hunk 2 above)
Comment (changab) [reply schema/virtual-memory-extension.asn:19]: sure, done in the second commit

**Review by timsifive — APPROVED** [review]

**Pull request merged**